g/guava-guava/guava-guava_ubi_9.3.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package       : guava-guava
+# Version       : v33.1.0
+# Source repo   : https://github.com/google/guava
+# Tested on     : UBI 9.3
+# Language      : Java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : kotla santhosh<kotla.santhosh@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+set -e
+
+REPO_NAME="guava"
```

Comment:
Dont need repo name and package name since they have the same value. Can use package name variable only for this case.